astropy/units/quantity.py

```diff
@@ -1722,3 +1722,61 @@ def _set_unit(self, unit):
                  ", so cannot set it to '{0}'.".format(unit)))
 
         super()._set_unit(unit)
+
+
+def isclose(a, b, rtol=1.e-5, atol=None, **kwargs):
```

Comment:
This now definitely is getting in the terrain of making more sense as a separate module/file! `quantity_helper` is an option, but perhaps better would be `quantity_utils`. But can be another PR.

Reply:
Yeah, let's defer moving it to separate module in a separate PR.